.github/workflows/tests.yml

```diff
@@ -0,0 +1,75 @@
+name: Tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron:  '0 0 * * 0'
+
+jobs:
+  standard-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build the project
+        run: |
+          echo "UID=$(id --user)" > .env
+          docker-compose up -d --build
+      - name: Lint the code
+        run: |
+          docker-compose exec -T caluma black --check .
+          docker-compose exec -T caluma flake8
+          docker-compose exec -T caluma ./manage.py makemigrations --check --dry-run --no-input
+          docker-compose exec -T caluma reuse lint
+      - name: Run pytest
+        run: |
```

Comment:
Not necessary, inline the following line

Reply:
Done